apps/etl-func/src/domain/message.ts

```diff
@@ -13,13 +13,18 @@ export const messageMetadataSchema = z.object({
   fiscalCode: z.string().min(1),
   id: z.string().ulid(),
   indexedId: z.string().ulid(),
-  isPending: z.boolean().optional(),
+  isPending: z.boolean(),
```

Comment:
@lucacavallaro as we discussed I removed the `optional` flag from isPending.
The function that calls `getMessageByMetadata` method will be responsible for ensuring that `isPending` is defined.